charts/internal/shoot-system-components/charts/calico-bgp/values.yaml

```diff
@@ -3,3 +3,4 @@ bgp:
   nodeToNodeMeshEnabled: false
   listenPort: 179
   bindMode: NodeIP
+  asNumber: true
```

Comment:
@defo89 would such a default value make sense here?

Reply:
not really, it's BGP autonomous system number which will be specific to the infrastructure. But we could use Calico default [64512](https://github.com/projectcalico/calico/blob/master/api/pkg/apis/projectcalico/v3/bgpconfig.go#L65-L66) here

Reply:
Thanks! I changed the value to `64512`. The issue was that the helm check is expecting that all `required` values are set in the `values.yaml`.